src/content/security/index.md

```diff
@@ -0,0 +1,215 @@
+---
+title: Ethereum security and scams
+description: Staying safe on Ethereum
+lang: en
+sidebar: true
+---
+
+## Staying Safe on Ethereum
+
+With interest in cryptocurrencies growing exponentially, learning best practices when using cryptocurrency is essential. Crypto can be fun and exciting, but there are also serious risks. If you put in this small amount of upfront work, you can mitigate these risks.
+
+<Divider />
+
+## Web Security 101
+
+### Use strong passwords
+
+Over 80% of account hacks are a result of weak or stolen passwords. A long combination of characters, numbers and symbols is best to keep your accounts secure.
+
+A common mistake individuals make is using a long combination of common dictionary words. Passwords like this are insecure because they are prone to a simple hacking technique known as a [dictionary attack](<[https://en.wikipedia.org/wiki/Dictionary_attack](https://en.wikipedia.org/wiki/Dictionary_attack)>).
+
+```md
+Example of a weak password: FluffyKittens!
+
+Example of a strong password: ymv\*azu.EAC8eyp8umf
+```
+
+#### Good password practices:
+
+- Make passwords **at least** 12 characters long
+- Use a mixture of uppercase, lowercase, numbers and symbols
+- Avoid common dictionary words
+
+[More on creating strong passwords](https://terranovasecurity.com/how-to-create-a-strong-password-in-7-easy-steps/)
+
+### Use unique passwords for everything
+
+A strong password doesn't provide as much protection if the password is revealed in a data breach. The website [Have I Been Pwned](<notion://www.notion.so/efdn/%5B%3Chttps://haveibeenpwned.com/%3E%5D(%3Chttps://haveibeenpwned.com/%3E)>) allows you to check if your accounts were involved in any data breaches stored in their database. If they have, **you should change pwned passwords immediately**. Using unique passwords for every account lowers the risk of hackers getting access to all of your accounts when one of your passwords is compromised.
+
+### Use a password manager
```

Comment:
Hmmm. If a user takes nothing else from this page other than downloading a password manager it's probably a win. I've added some emphasis to this section to push that.

![Screenshot 2021-09-14 at 11 25 15](https://user-images.githubusercontent.com/62268199/133241505-ce702ae7-571a-4a82-85c7-b2794acc673a.png)

At the same time, I think going through this should be a linear journey. (i.e. IRL people learn to use symbols and numbers before they make the jump to password managers). Unfortunately, a lot of people won't download a pw manager because of the overhead, I'm not sure moving this to position #1 helps that.

I'm kinda on the fence here - can we discuss this more at standup @wackerow? Would love some other opinions on this.

cc: @lukassim @pettinarip @corwintines @samajammin

Reply:
I think you both have good points here. Will do some thinking about it.

Reply:
@minimalsm Since you've mentioned the costs connected with password managers, would it also make sense to mention which ones offer free plans? I know LastPass and Bitwarden do, while KeePass seems to be completely free.

Reply:
> Since you've mentioned the costs connected with password managers, would it also make sense to mention which ones offer free plans? I know LastPass and Bitwarden do, while KeePass seems to be completely free.

@lukassim sorry I meant cognitive overhead — “how many logical connections or jumps your brain has to make in order to understand or contextualize the thing you’re looking at.”

But good idea we should work out a fair way to differentiate them based on price.